packages/management/tanzu-auth/bundle/config/deployment.yaml

```diff
@@ -0,0 +1,22 @@
+#@ load("@ytt:data", "data")
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: tanzu-auth-controller-manager
+  namespace: tanzu-auth
+spec:
+  selector:
+    matchLabels:
+      app: tanzu-auth-controller-manager
+  template:
+    metadata:
+      labels:
+        app: tanzu-auth-controller-manager
+    #! TODO: check out the spec of other packages, such as addons-manager-v1 for resource limits, etc.
+    #! we probably want to flesh this out.
+    spec:
+      serviceAccountName: tanzu-auth-controller-manager-sa
+      containers:
+      - image: #@ data.values.image
```

Comment:
issue: i see this `data.values.image` here, but i don't see it in the `values.yaml` file; do we need to add it?

Reply:
so, oddly, it works.... which is kind of weird?

Reply:
we pass it through in `hack/run.sh`:
```
ytt --data-value "image=$image" -f "${TANZU_AUTH_CONFIG_DIR}" | kbld -f - --imgpkg-lock-output "${TANZU_AUTH_BUNDLE_DIR}/.imgpkg/images.yaml" | kapp deploy -a "${DEPLOYMENT_NAME}" -f - -y
```

Reply:
per our convo, I believe it works because we don't have a `schema.yaml` enforcing what can and cannot be passed.
That said, I'll add this field to `values.yaml`.